cfgov/unprocessed/css/organisms/mega-menu.less

```diff
@@ -5,46 +5,488 @@
+                                    </div>
+                                    <aside class="sub-nav_media">
+                                        <a href="/the-bureau/">
+                                            <img src="/static/img/nav-about-us-video.png" alt="About Us video image">
+                                            <p class="h4 u-mt15">
+                                              The CFPB: Working for you.
+                                            </p>
+                                        </a>
+                                        <p>
+                                            This short video covers what the CFPB is and
+                                            how we are working for American consumers.
+                                        </p>
+                                    </aside>
+                                </div>
+                            </div>
+                        </div>
+                    </li>
+                </ul>
+                [Global Eyebrow]
+            </nav>
         </div>
       codenotes:
         - |
           Structural cheat sheet:
           -----------------------
           .o-mega-menu
+            .o-mega-menu_trigger
+            .o-mega-menu_content
+                ul.o-mega-menu_content-list
+                    li.o-mega-menu_content-item
+                        .m-global-header-cta.m-global-header-cta__list
+                    li.o-mega-menu_content-item
+                        a.o-mega-menu_content-link
+                    li.o-mega-menu_content-item
+                        a.o-mega-menu_content-link
+                        .o-mega-menu_subcontent
+                            .o-mega-menu_subcontent-wrapper
+                                .wrapper
+                                    button.o-mega-menu_subcontent-btn__back
+                                    .o-mega-menu_subcontent-grid
+                                        .o-mega-menu_subcontent-title
+                                            a.o-mega-menu_subcontent-link
+                                        .o-mega-menu_subcontent-lists
+                                            ul.o-mega-menu_subcontent-list
+                                                li.o-mega-menu_subcontent-item
+                                            ul.o-mega-menu_subcontent-list
+                                                li.o-mega-menu_subcontent-item
+                                    aside.sub-nav_media
   tags:
     - cfgov-organisms
 */
 
 .o-mega-menu {
 
-    .m-global-header-cta {
+    &_content-item,
+    &_subcontent-item {
+        // Override margin added by CF to list items.
+        margin-bottom: 0;
+    }
 
-        .respond-to-min( @bp-med-min, {
-            display: none;
-        } );
+    &_content {
+        &-link {
+            // Colors for :link, :visited, :hover, :focus, :active.
+            .u-link__colors( @black, @black, @black, @black, @black );
```

Comment:
Sweet! Fixed in https://github.com/cfpb/cfgov-refresh/commit/83f76dc218a3402b12cee0251154849a6aea7a68